src/integration-test/kotlin/integration/eth/EthRelayProviderIrohaTest.kt

```diff
@@ -116,8 +119,8 @@ class EthRelayProviderIrohaTest {
                 {
                     assert(it.isEmpty())
                 },
-                {
-                    fail { "result has exception $it" }
+                { ex ->
+                    fail("result has exception", ex)
```

Comment:
Is there any reason to change brackets ?

Reply:
it has two arguments: short message and exception.